### PR TITLE
Resolves #2589

### LINKED ISSUE #2589: Missing information for start-bitstransfer
The document completely ignores a parameter

`Start-BitsTransfer -SecurityFlags`

`EnableCRLCheck,                  IgnoreCertWrongUsage,            RedirectPolicyAllowReport,
IgnoreCertCNInvalid,             IgnoreUnknownCA,                 RedirectPolicyAllowSilent,
IgnoreCertDateInvalid,           RedirectPolicyAllowHttpsToHttp,  RedirectPolicyDisallow`

there no explanation here or in `get-help Start-BitsTransfer -full`
this includes powershell 7/6/5, I have not checked PS4

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 70fb39b0-6b7e-a143-683b-5966b872c938
* Version Independent ID: 63016f51-2ccf-fdb0-5e29-16418508ce64
* Content: [Start-BitsTransfer (BitsTransfer)](https://docs.microsoft.com/en-us/powershell/module/bitstransfer/start-bitstransfer?view=windowsserver2019-ps&source=docs)
* Content Source: [docset/winserver2019-ps/bitstransfer/Start-BitsTransfer.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/bitstransfer/Start-BitsTransfer.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

### PR DESCRIPTION
**Pull request by velkovb**
Add parameter for Windows 10.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2589

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by dariomws** [comment]
@velkovb are you sure that this parameter is required?

**Comment by velkovb** [comment]
@dariomws Good catch. It is now required. I marked it in the syntax section as not required but have missed the description.

File: docset/winserver2019-ps/bitstransfer/Start-BitsTransfer.md [hunk 1]
```diff
@@ -486,6 +486,24 @@ Accept pipeline input: False
...
+
+Note: This parameter is only available in Windows 10.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+Accepted values: RedirectPolicyAllowSilent, EnableCRLCheck, IgnoreCertCNInvalid, IgnoreCertDateInvalid, IgnoreUnknownCA, IgnoreCertWrongUsage, RedirectPolicyAllowReport, RedirectPolicyDisallow, RedirectPolicyAllowHttpsToHttp
+
+Required: True
```
Comment (velkovb) [inline docset/winserver2019-ps/bitstransfer/Start-BitsTransfer.md:500]:
```suggestion
Required: False
```

**Review by dariomws — APPROVED** [review]

**Comment by e0i** [comment]
@tiburd The PR has been approved by the reviewers and is ready to be merged. Thanks.

**Pull request merged**